build-tools/automation/azure-pipelines-apidocs.yaml

```diff
@@ -0,0 +1,70 @@
+# Pipeline for updating and uploading android-api-docs/docs/Mono.Android/en/*
+# https://dev.azure.com/devdiv/DevDiv/_build?definitionId=15262
+
+trigger: none
+pr: none
```

Comment:
So is the plan to just run this every now and then? Should we make a document that says how to do it?

Reply:
Yeah, my thinking was that every time we get a new binding version or fix something in the javadoc transformation we could use this to build a new set of docs. Eventually the pipeline could probably create a PR into android-api-docs, but I don't think that's necessary for now.